### DISCUSSION TITLE
如何保存实况图

### DISCUSSION THREAD

**Discussion by KLimerencer** [Evil0ctal/Douyin_TikTok_Download_API#506] [Q&A]
实况和图片在一起的视频可以解析了但是只能全都存为图片格式 怎么能保存实况呢 我搜了很多解析抖音的目前就只有机领网的可以下载大部分实况（有部分依旧只能保存为图片）但不知道那个怎么做到的

**Comment by Evil0ctal** [comment]
你发几个视频的分享链接，我去研究一下。

**Reply by KLimerencer** [reply]
9.99 04/26 QXZ:/ G@v.sE 拧巴的人 需要一个赶不走的爱人  https://v.douyin.com/iD14Lr6j/ 复制此链接，打开Dou音搜索，直接观看视频！
